principles/fp-019-term-stability.md

```diff
@@ -68,10 +69,20 @@ Note that some older implementations in OWL used the CURIE method as shown below
-  -  OBO: Use `relationship:` with the CURIE for the annotation property (IAO:0000231) and a CURIE for the specific reason (an individual from the "obsolescence reason specification" term [IAO:0000225](http://purl.obolibrary.org/obo/IAO_0000225)). See below for example. Note that older implementations often used alternative methods (described after the examples). These methods are still valid, but are not preferred.
+7. Prepend the string "OBSOLETE. " (this precise string, including the space) to the term definition. NOTE: This MUST be implemented consistently. That is, if applied at all, it has to be applied to every obsoleted term definition.
+8. Indicate the reason(s) for obsoleting:
+   -  OWL: Use the `has obsolescence reason` annotation property from OMO ([IAO:0000231](http://purl.obolibrary.org/obo/IAO_0000231)) with the value set to the IRI of one of the individuals of the "obsolescence reason specification" term [IAO:0000225](http://purl.obolibrary.org/obo/IAO_0000225). See below for example.
+   -  OBO: Use `relationship:` with the CURIE for the annotation property (IAO:0000231) and a CURIE for the specific reason (an individual from the "obsolescence reason specification" term [IAO:0000225](http://purl.obolibrary.org/obo/IAO_0000225)). See below for example. Note that older implementations often used alternative methods (described after the examples). These methods are still valid, but are not preferred.
+
+
+|IRI            |CURIE      |Label                  |Definition|Editor note    |
+|---------------|-----------|-----------------------|----------|-------------- |
+|obo:IAO_0000103|IAO:0000103|failed exploratory term|The term was used in an attempt to structure part of the ontology but in retrospect failed to do a good job|???|
+|obo:IAO_0000226|IAO:0000226|placeholder removed    |???   | ???               |
+|obo:IAO_0000227|IAO:0000227|terms merged |???|An editor note should explain what were the merged terms and the reason for the merge.                               |
+|obo:IAO_0000228|IAO:0000228|term imported|???|This is to be used when the original term has been replaced by a term imported from an other ontology. An editor note should indicate what is the URI of the new term to use.|
+|obo:IAO_0000229|IAO:0000229|term split   |???|This is to be used when a term has been split in two or more new terms. An editor note should indicate the reason for the split and indicate the URIs of the new terms created.|
+|obo:OMO_0001000|OMO:0001000|out of scope |???|This obsolesence reason should be used conservatively. Typical valid examples are: un-necessary grouping classes in disease ontologies, a phenotype term added on the assumption it was a disease.|
```

Comment:
While this is highly useful--and we agree it should be documented--we don't believe it should be on this principles page since it is more detailed than we'd like. There is a page (the OBO Academy "O-book" on obsoletion) that we already reference under Implementation (first sentence below that heading) which is intended to provide such details. We will look into having this information added there and, once done, will highlight that page again under the section mentioning the obsolescence reasons. We see that the same request was made under the ontology-metadata repo, so the information won't be lost. We thus ask that the table above be removed from your suggested changes.

Reply:
Fine. Implemented in https://github.com/OBOAcademy/obook/pull/546